packages/fast-components-react-base/src/listbox/listbox.props.ts

```diff
@@ -43,6 +43,16 @@ export interface ListboxHandledProps extends ListboxManagedClasses {
      * The  onSelectedItemsChanged event handler
      */
     onSelectedItemsChanged?: (selectedItems: ListboxItemProps[]) => void;
+
+    /**
+     * Whether a listitem should automatically get focus when this component is mounted
+     */
+    autoFocus?: boolean;
```

Comment:
I think this may just be the terminology, but I believe autoFocus on a `select` would actually focus the control: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/autofocus

Reply:
Renamed to "focusItemOnMount"